### PR TITLE
Clarify how email courtesy copies now work

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/pZqhXafv/671-update-template-for-individual-type-emails-inc-one-click-unsubscribe

### CODE REVIEW

File: source/manual/email-notifications-how-they-work.html.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ The email notification system generates 3 different types of email. These are tr
 - [Technical dashboard][email-alert-api-technical]
 - [Product dashboard][email-alert-api-product]
 - [How content is matched to subscriber lists][email-alert-api-matching]
-- [Courtesy copies Google group][google-group] - provides a way to view replicas of every immediate email notification generated in production
+- [Courtesy copies Google group][google-group]
+  - Provides a way to view replicas of every immediate email notification generated in production.
+  - Works by the group being subscribed to the "all-government-publishing-updates" list ([here](courtesy-subscription-page)).
```
Comment (kevindew) [inline source/manual/email-notifications-how-they-work.html.md:33]:
```suggestion
  - Works by the group being subscribed to the "all-government-publishing-updates" list ([here][courtesy-subscription-page]).
```

File: source/manual/email-notifications-how-they-work.html.md [hunk 2]
```diff
@@ -65,6 +67,7 @@ Communication from Email Alert API to Notify is done via a HTTP API which is aut
 [email-alert-api-matching]: https://github.com/alphagov/email-alert-api/blob/master/docs/matching-content-to-subscriber-lists.md
 [receiving-integration-staging-email]: /manual/receiving-emails-from-email-alert-api-in-integration-and-staging.html
 [google-group]: https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/#!forum/govuk-email-courtesy-copies
+[courtesy-subscription-page]: https://www.gov.uk/search/all/email-signup
```
Comment (kevindew) [inline source/manual/email-notifications-how-they-work.html.md:70]: > You’ll get emails when we add or update pages about:
>
> Search

Oh dear 😬

**Review by kevindew — APPROVED** [review]
Looks good - just a link to fix.

File: source/manual/email-notifications-how-they-work.html.md (hunk 1 above)
Comment (benthorner) [reply source/manual/email-notifications-how-they-work.html.md]: Oops - fixed 👍

**Pull request merged**